src/test/unit/regions/TMRegionTest.cpp

```diff
@@ -328,7 +328,7 @@ TEST(TMRegionTest, testLinking) {
   EXPECT_EQ(r4InputArray, expected3outa) << r4InputArray;
 
   // cleanup
-  region3->executeCommand({"closeFile"});
+  region4->executeCommand({"closeFile"});
```

Comment:
I accidentaly discovered this bug in test. Closing file was for TMRegion but should be for FileOutputRegion obviously. Before there was no code in TMRegion ExecuteCommand() so nothing was thrown.

Reply:
Another good catch. :)

Reply:
Hmmm, maybe it should throw an exception if a command is requested but it is not implemented.
In RegionImpl.cpp line 210
```
std::string RegionImpl::executeCommand(const std::vector<std::string> &args, Int64 index) {
  // This Region does not execute any Commands.
  return "";
}
```
Should be something like ...
```
std::string RegionImpl::executeCommand(const std::vector<std::string> &args, Int64 index) {
  std::string command;
  if (args.size() > 0)
    command = args[0];

  // This Region does not execute any Commands.
  NTA_THROW << getName() << ": Does not implement the command  '" << command << "'";
}```

Can you include this in your PR  (along with a simple unit test)?

Reply:
Ok, sure. It is there